yt_dlp/extractor/microsoftembed.py

```diff
@@ -0,0 +1,80 @@
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+    unified_timestamp,
+    int_or_none,
+)
+from collections import OrderedDict
+
+
+class MicrosoftEmbedIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?microsoft\.com/en-us/videoplayer/embed/(?P<id>[a-z0-9A-Z]+)'
+
+    _TESTS = [{
+        'url': 'https://www.microsoft.com/en-us/videoplayer/embed/RWL07e',
+        'info_dict': {
+            'id': 'RWL07e',
+            'title': '...'
+        }
+    }]
```

Comment:
Fill this out fully till the download test passes

Reply:
filled more test info in commit aa3d71880f2bf8d5047c6e739e814b6a0aa9ca31

Reply:
tests pass in 64ae790df1ca3bb4c4c23a9999f4e8ceedd6640f